fsspec/implementations/smb.py

```diff
@@ -193,16 +204,24 @@ def _open(
         """
         block_size: int or None
             If 0, no buffering, 1, line buffering, >1, buffer that many bytes
+
+        Notes
+        -----
+        By specifying 'share_access' in 'kwargs' it is possible to override the
+        default shared access setting applied in the constructor of this object.
         """
         bls = block_size if block_size is not None and block_size >= 0 else -1
         wpath = _as_unc_path(self.host, path)
+        share_access = kwargs.pop("share_access", self.share_access)
         if "w" in mode and autocommit is False:
             temp = _as_temp_path(self.host, path, self.temppath)
             return SMBFileOpener(wpath, temp, mode, block_size=bls, **kwargs)
-        return smbclient.open_file(wpath, mode, buffering=bls, **kwargs)
+        return smbclient.open_file(
+            wpath, mode, buffering=bls, share_access=share_access, **kwargs
+        )
 
     def copy(self, path1, path2, **kwargs):
-        """ Copy within two locations in the same filesystem"""
```

Comment:
This edit is a result of running "black" against the file, due to the linting failures shown on a previous build. I'm a bit unsure about it since this isn't a line I changed, so not sure why linting wasn't a problem already on master.

Reply:
Maybe a new version of `black`